pandas/core/dtypes/dtypes.py

```diff
@@ -558,7 +558,7 @@ def validate_categories(categories, fastpath: bool = False):
         if not fastpath:
 
             if categories.hasnans:
-                raise ValueError("Categorial categories cannot be null")
+                raise ValueError("Categorical categories cannot be null")
```

Comment:
it looks like tests are failing because this same typo exists elsewhere in the tests, so needs to be fixed there too

Reply:
You're right. It was a commit from another branch I did. That one has been merged as of this morning, so it should go through alright now.